.github/workflows/push-to-gcr.yml

```diff
@@ -0,0 +1,24 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v4
+    - name: Build the Docker image
+      run: docker build . --file Dockerfile --tag gcr.io/oss-fuzz-base/oss-fuzz-gen:daily
+    - name: Authenticate to gcloud
+      uses: google-github-actions/auth@v2
+      with:
+        credentials_json: '${{ secrets.GCLOUD_CREDENTIAL }}'
+    - name: Set up Cloud SDK
+      uses: google-github-actions/setup-gcloud@v2
+    - name: Push to GCR
+      run: gcloud docker -- push gcr.io/oss-fuzz-base/oss-fuzz-gen:daily
```

Comment:
@oliverchang Just to double check: Would we prefer continue using this base registry, or the artifact registry?

Reply:
Artifact registry might be easier for us to accept external images when they request help with experiments.

Reply:
@DonggeLiu do you want to merge this and come back to this discussion later?

Reply:
Could we use `us-central1-docker.pkg.dev/oss-fuzz-base/testing/<image-name>`?

Reply:
Don